.buildkite/pipeline.full.yml

```diff
@@ -268,12 +346,198 @@ steps:
+    depends_on:
+      - "fixture-r19"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/release/fixture-r19.apk"
+      docker-compose#v3.7.0:
+        run: android-maze-runner
+        command:
+          - "features/full_tests/flaky.feature"
+          - "--retry=2"
+          - "--strict-undefined"
+          - "--strict-pending"
+          - "--app=/app/build/release/fixture-r19.apk"
+          - "--farm=bs"
+          - "--device=ANDROID_7_1"
+    concurrency: 9
+    concurrency_group: 'browserstack-app'
+
+  - label: ':hankey: Flaky Android 8.0 NDK r19 end-to-end tests'
+    depends_on:
+      - "fixture-r19"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/release/fixture-r19.apk"
+      docker-compose#v3.7.0:
+        run: android-maze-runner
+        command:
+          - "features/full_tests/flaky.feature"
+          - "--retry=2"
+          - "--strict-undefined"
+          - "--strict-pending"
+          - "--app=/app/build/release/fixture-r19.apk"
+          - "--farm=bs"
+          - "--device=ANDROID_8_0"
+    concurrency: 9
+    concurrency_group: 'browserstack-app'
+    soft_fail:
+      - exit_status: "*"
+
+  - label: ':hankey: Flaky Android 8.1 NDK r19 end-to-end tests'
+    depends_on:
+      - "fixture-r19"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/release/fixture-r19.apk"
+      docker-compose#v3.7.0:
+        run: android-maze-runner
+        command:
+          - "features/full_tests/flaky.feature"
+          - "--retry=2"
+          - "--strict-undefined"
+          - "--strict-pending"
+          - "--app=/app/build/release/fixture-r19.apk"
+          - "--farm=bs"
+          - "--device=ANDROID_8_1"
+    concurrency: 9
+    concurrency_group: 'browserstack-app'
+    soft_fail:
+      - exit_status: "*"
+
+  - label: ':hankey: Flaky Android 10 NDK r21 end-to-end tests'
+    depends_on:
+      - "fixture-r21"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/release/fixture-r21.apk"
+      docker-compose#v3.7.0:
+        run: android-maze-runner
+        command:
+          - "features/full_tests/flaky.feature"
+          - "--retry=2"
+          - "--strict-undefined"
+          - "--strict-pending"
+          - "--app=/app/build/release/fixture-r21.apk"
+          - "--farm=bs"
+          - "--device=ANDROID_10_0"
+    concurrency: 9
+    concurrency_group: 'browserstack-app'
+
+  - label: ':hankey: Flaky Android 11 NDK r21 end-to-end tests'
+    depends_on:
+      - "fixture-r21"
+    timeout_in_minutes: 60
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/release/fixture-r21.apk"
+      docker-compose#v3.7.0:
+        run: android-maze-runner
+        command:
+          - "features/full_tests/flaky.feature"
+          - "--retry=2"
+          - "--strict-undefined"
```

Comment:
Perhaps a future enhancement, but if there's any way of specifying this (and other flags) via mazerunner config/in one place it would help make the yaml a bit more readable

Reply:
Yes, I agree and already an idea in the back of my mind and an enhancement (maybe more than one) to MR.